extensions/amp-list/amp-list.md

```diff
@@ -261,6 +261,35 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 
 If `binding` attribute is not provided, default is `always`.
 
+## Experimental: Infinite Scroll (amp-list-load-more)
+We've introduced an experiment called `amp-list-load-more` as an implementation for pagination and infinite scroll in amp-list. This is an experimental feature, and final APIs may change.
+
+### Attributes
+#### load-more (mandatory)
+Adding this attribute will allow amp-list (with no value) to show a “load-more” button at the end of the amp-list. The value of this attribute can be set to “auto” to trigger automatic loading more elements three viewports down for an infinite scroll effect.
+
+#### load-more-bookmark (mandatory)
+This attribute specifies an attribute in the returned data that will give the url of the next items to load. E.g. In the following sample payload, we would specify `load-more-bookmark=”next”`.
+
+```
+{ “items”: [], “next”: “https://url.to.load” }
```

Comment:
nit: You should replace these smart-quotes `“”` with regular quotes `""`, so people who copy-paste from the documentation don't need to correct them after they paste

Reply:
Yup, awkward copy-paste from google docs.